src/java.base/macosx/native/libnio/fs/BsdNativeDispatcher.c

```diff
@@ -261,7 +279,8 @@ Java_sun_nio_fs_BsdNativeDispatcher_setattrlist0(JNIEnv* env, jclass this,
     attrList.bitmapcount = ATTR_BIT_MAP_COUNT;
     attrList.commonattr = commonattr;
 
-    if (setattrlist(path, &attrList, (void*)buf, count*attrsize, options) != 0) {
+    if (setattrlist(path, &attrList, (void*)buf,
+        count*attrsize, options) != 0) {
```

Comment:
I assume this is an accidental edit.

Reply:
Right; fixed in 2157a1480229fa697c824ae814f11a309627d768.